cmd/tsbs_load_clickhouse/profile.go

```diff
@@ -0,0 +1,49 @@
+package main
```

Comment:
So I know we kind of started this pattern by putting the `profile` stuff in `tsbs_load_timescaledb`, but now that we're adding general system profiling to more of the databases, I'm wondering if it might make sense to extract this to a generic package that could be re-used by all of the databases, instead of having to duplicate OS profiling calls for each separate DB that we implement?

Reply:
There's actually a few problems with this profiling, the big one being that it doesn't work if the client & server are not the same. Given that, I think it's probably better to just not include this in new databases (i.e., remove this file and its other code) until we come up with a cleaner way to do this. It probably should be removed from TimescaleDB too but I haven't bothered to do it yet.